apps/react/src/challenges/feedback-modal/components/RatingContent.jsx

```diff
@@ -0,0 +1,22 @@
+import content from '../data/emojicontent';
+import { appendActiveClass } from '../utilities/ClassUtility';
+import styles from '../styles.module.css';
+
+const RatingContent = ({ handleClick, activeRating }) => {
+  return (
+    <div className={styles.iconContainer}>
+      {content.map((item) => (
+        <div
+          key={item.rating}
+          className={appendActiveClass(item.rating, activeRating, styles.iconItem)}
+          onClick={() => handleClick(item)}
+        >
+          <img src={item.url} className={styles.iconImage} alt={item.mood} />
+          <p>{item.mood}</p>
+        </div>
+      ))}
```

Comment:
How about an input type "radio" group for the ratings. You can put the radio buttons inside a &lt;form&gt;.

Reply:
@arpansaha13 Hi Arpan, that is definitely doable, but I took the inspiration from an online masterclass that I had taken some time back. They had used emojis. Changing it to a radio group would kill the look and feel of it. I don't know if I am making sense, but please do let me know, if needed I will change it. Also, I hope I was able to understand what you are trying to imply.

Reply:
@arpansaha13 Please let me know your thoughts on this.

Reply:
Yeah it's my mistake I missed some things...

What I was thinking was... an invisible or hidden radio button for each emoji... and the actual "div" can be inside a "label" associated with that radio button.

```jsx
         // this radio button is hidden
        <input type="radio" id="first_option" name="feedback_rating" value="0">
        <label
          for="first_option"
          key={item.rating}
          className={appendActiveClass(item.rating, activeRating, styles.iconItem)}
          onClick={() => handleClick(item)}
        >
          <img src={item.url} className={styles.iconImage} alt={item.mood} />
          <p>{item.mood}</p>
        </label>
```

In this way the labels can display the emoji and radio button won't show.

Reply:
@arpansaha13 Thank you, will work on it by today or tomorrow and update the changes.